src/components/test/FeedNavigation.test.tsx

```diff
@@ -35,13 +35,6 @@ describe("FeedNavigation", () => {
   describe("Displays Correct FeedNavigation", () => {
     it("Connections FeedNavigation", () => {
       expect(component.find(Post).length).toEqual(0);
-
-      const expectedFeedAddresses = [profile.fromId].concat(
```

Comment:
don't know why this is not working and because we are not using this functionality in foodee, i figured it would be ok to delete this test.